_posts/2020-08-26-#19339.md

```diff
@@ -0,0 +1,94 @@
+When a node receives a transaction from its peers, it uses an empty value for `nAbsurdFee` when calling ATMP
+(you can see the values of `nAbsurdFee` removed in [commit]()). 
+Furthermore, there is no way for a node operator to set the maximum fee, e.g. through a command line argument on start.
+
+* [#19339](https://github.com/bitcoin/bitcoin/pull/19339) redelegates fee checking from the mempool to the clients in order to make mempool logic less user-specific. 
+The main observable behavior change is in the error message, which is updated to include RPC and help reviewers confirm 
+the transition from validation logic to transaction error.
+
+  *  Note that this only changes RPCs [`testmempoolaccept`](https://github.com/bitcoin/bitcoin/blob/3ab2582c7fe76d2839ab493512758d5601903c86/src/rpc/rawtransaction.cpp#L858) 
+  and [`sendrawtransaction`](https://github.com/bitcoin/bitcoin/blob/3ab2582c7fe76d2839ab493512758d5601903c86/src/rpc/rawtransaction.cpp#L800).
+     There are no changes to wallet (reviewers may want to confirm that this is ok).
+     
+  * Note that clients often use [`BroadcastTransaction()`](https://github.com/bitcoin/bitcoin/blob/3ab2582c7fe76d2839ab493512758d5601903c86/src/node/transaction.cpp#L16), 
+    to interface with ATMP; it also takes `max_tx_fee` to pass in as `nAbsurdFee`. 
+    This PR adds fee checking to `BroadcastTransaction()` by adding a dry run of ATMP before the real submission.
+
+* To be careful to preserve the fee protection offered to the user, the PR includes a few intermediate stages. 
+The author recommends reviewing this PR by checking out each commit individually and confirming that the behavior is 
+identical at each stage.
+
+  (1) [`18f7e1f`](https://github.com/bitcoin/bitcoin/pull/19339/commits/18f7e1fb2dc1c66c2f3ac916709aeec12252861b) 
+  Catch mempool rejections based on `AbsurdFee` and return `MAX_FEE_EXCEEDED` instead of 
+  `TX_NOT_STANDARD` for high fees. 
+
+  (2) [`a872d8d`](https://github.com/bitcoin/bitcoin/pull/19339/commits/a872d8d31bcc2dd1089cb9fd68a48b3d798bd59b) 
+  In `BroadcastTransaction`, first dry-run ATMP and check the fee that is returned. 
+  If the fee is below `max_tx_fee`, send the transaction for real.
+
+  (3) [`2409462`](https://github.com/bitcoin/bitcoin/pull/19339/commits/24094623b5c85688e95641c269275d069fb50d2a) Ignore `nAbsurdFee` in ATMP.
+
+  (4) [`67bb714`](https://github.com/bitcoin/bitcoin/pull/19339/commits/67bb7148ad4142611ab132047548f96dd3d59428) Remove `nAbsurdFee` from ATMP.
+
+
+## Questions
+
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or
+   NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+
+2. Does this PR change any behavior or is it a pure refactor?
+
+3. There was some discussion as to whether AbsurdFee is policy or a client preference. 
+What do you think, and how did you determine this?
+
+4. Compare RPCs `testmempoolaccept` and `sendrawtransaction`’s implementations. 
```

Comment:
suggest: s/RPCs x and y's implementations/the implementations of the x and y RPCs/